torch/fx/passes/runtime_assert.py

```diff
@@ -1,7 +1,11 @@
 # mypy: allow-untyped-defs
+import contextlib
```

Comment:
Is attempting to look at the diff here at all useful, or should I just read it as an entirely new file?